src/txn/txn_rollback_to_stable.c

```diff
@@ -1087,8 +1090,11 @@ __rollback_to_stable_btree_hs_truncate(WT_SESSION_IMPL *session, uint32_t btree_
          * we can skip it.
          */
         if (__wt_txn_visible_all(
-              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts))
+              session, cbt->upd_value->tw.stop_txn, cbt->upd_value->tw.durable_stop_ts)) {
+            WT_STAT_CONN_INCR(session, cursor_prev_hs_tombstone);
+            WT_STAT_DATA_INCR(session, cursor_prev_hs_tombstone);
```

Comment:
Are you sure that this is going to be incrementing for the data store's dhandle (as opposed to the history store table)?

Reply:
Good point, occasionally we do run these operations with the history store dhandle as opposed to the data store one as such i'll remove it from that.